### DISCUSSION TITLE
Difficulty to see clearly Application Flows

### DISCUSSION THREAD

**Discussion by AlgamEric** [dbarzin/mercator#257] [General]
Hello, 
Thank you for this greatfull tool !

Is it possible to obtain a clearer view  of the Application Flows ?
Because the final result is too condensed and it's hard to read.

![Vue des flux applicatifs](https://user-images.githubusercontent.com/117906009/201065056-68c53b28-f9d8-4b53-8c4e-c000e6c9d028.png)

**Comment by dbarzin** [comment]
You may use the "explorer" available under "documentation" -> "explore" to get a better view.
It could generate a better graph with you applications and flows if you filter on "Applications"

![image](https://user-images.githubusercontent.com/43208232/201078691-f87bdc80-1160-40f5-af4f-33356f215a94.png)

See here with the "Physical view" : 
[Image](https://github.com/dbarzin/mercator/blob/master/public/screenshots/mercator9.png)

It is available since few days, you may have to update Mercator.

**Reply by AlgamEric** [reply]
Thank you for your quick answer. I'll tell you after the update.

**Comment by AlgamEric** [comment]
Bonjour, en effet la vue "Documentation / Exploration" est beaucoup plus sympa, merci ! 
![image](https://user-images.githubusercontent.com/117906009/206389351-29e470e0-ef67-4e2f-afce-ed92ea6c60a5.png)

Par contre sur cette vue, nous n'avons plus la possibilité de rentrer dans le détail des flux, des entités, ...
=> Est-il envisagé d'avoir aussi cette fonctionnalité sur cette vue ?
